frontends/ast/simplify.cc

```diff
@@ -3337,6 +3337,15 @@ skip_dynamic_range_lvalue_expansion:;
 						wire->type = AST_LOCALPARAM;
 						wire->attributes.erase(ID::nosync);
 						wire->children.insert(wire->children.begin(), arg->clone());
+						// args without a range implicitly have width 1
+						if (wire->children.back()->type != AST_RANGE) {
+							AstNode* range = new AstNode();
+							range->type = AST_RANGE;
+							wire->children.push_back(range);
+							std::vector<RTLIL::State> data{{}};
```

Comment:
What does this vector initialize to, exactly? A single `S0` element?

Reply:
I'm using this to create constants of value `1'b0` for use in creating an implicit range such as `[0:0]`. Is there a cleaner way to accomplish this?

Reply:
`RTLIL::Const(0)`?